catalyst/utils/torch.py

```diff
@@ -343,6 +343,18 @@ def trim_tensors(tensors):
     return tensors
 
 
+def normalize(samples: Tensor) -> Tensor:
+    """
+    Args:
+        samples: tensor with shape of [n_samples, features_dim]
+    Returns:
```

Comment:
we need a new line before the `Returns` :)

Reply:
ok, done